test/unit/util.test.js

```diff
@@ -640,7 +641,7 @@ describe('SCHEMA UTILITY FUNCTION TESTS ', function () {
       expect(retVal).to.be.an('array');
       expect(retVal[0].key).to.equal('varName');
       expect(retVal[0].description).to.equal('varDesc');
-      expect(retVal[0].value).to.be.a('number');
+      expect(retVal[0].value).to.be.a('string');
```

Comment:
why should it be a string if type is integer?

Reply:
As all params are serialised value will be string and not a number. Also collection v2.x schema expect value to be string.